mumd/src/audio/output.rs

```diff
@@ -183,7 +183,7 @@ impl DefaultAudioOutputDevice {
             .supported_output_configs()
             .map_err(|e| AudioError::NoConfigs(AudioStream::Output, e))?
             .find_map(|c| {
-                if c.min_sample_rate() <= sample_rate && c.max_sample_rate() >= sample_rate {
+                if c.min_sample_rate() <= sample_rate && c.max_sample_rate() >= sample_rate && c.channels() == 2 {
```

Comment:
This forces output to be stereo, which can cause problems if the device does not support stereo. Maybe make it so that it prefers stereo, but will fall back to mono if it does not support stereo?

Reply:
#108

Reply:
Yeah sure.